### PR TITLE
Update readme add some description

### PR DESCRIPTION
**Pull request by arvryna**
* Removed empty dockerfile, which seems to confuse when someone wants to
  build the project
* Fixed - https://github.com/requiemofthesouls/pigeomail/issues/8

### CODE REVIEW

**Review by requiemofthesouls — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 # Pigeomail
-Service which provides securely personal email addresses written in pure Go.
+
+- Service which provides securely personal email addresses written in pure Go.
+- using this service, through our telegram bot, one can create an email, receive incoming emails.
```
Comment (requiemofthesouls) [inline README.md:4]:
```suggestion
- Using this service, through our telegram bot, one can create an email, receive incoming emails.
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+- currently you can only recieve emails, but in future we will add sending emails via the bot as well
```
Comment (requiemofthesouls) [inline README.md:5]:
```suggestion
- Currently you can only recieve emails, but in future we will add sending emails via the bot as well
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+- we don't store the emails in the server, but we will keep them in rmq until the user opens the bot to receive those emails.
```
Comment (requiemofthesouls) [inline README.md:6]:
```suggestion
- We don't store the emails in the server, but we will keep them in rmq until the user opens the bot to receive those emails.
```

**Comment by arvryna** [comment]
yes, Dockerfile is already removed

**Comment by arvryna** [comment]
pushed review changes

**Comment by requiemofthesouls** [comment]
> yes, Dockerfile is already removed

I mean restore it, cause the directory will be removed too

**Comment by arvryna** [comment]
> > yes, Dockerfile is already removed
> 
> I mean restore it, cause the directory will be removed too

ah okay)

**Review by requiemofthesouls — APPROVED** [review]
alright, ready to be merged

**Pull request merged**